packages/mds-agency/tests/integration-tests.ts

```diff
@@ -362,7 +334,7 @@ describe('Tests API', () => {
       .expect(201)
       .end((err, result) => {
         log('err', err, 'body', result.body)
-        test.string(result.body.result).contains('success')
```

Comment:
As per the [Agency Vehicles](https://github.com/openmobilityfoundation/mobility-data-specification/tree/dev/agency#vehicle---register) spec, we should be returning nothing along with the 201

Reply:
Minus the version, of course.